apps/styleguide/src/mixins/TableMetadataMixin.vue

```diff
@@ -36,7 +36,16 @@ export default {
       this.graphqlError = null;
       request(
         this.graphqlURL,
-        "{_session{email,roles}_schema{name,tables{name,tableType,id,description,externalSchema,semantics,columns{name,id,columnType,key,refTable,refLink,refLabel,refBack,required,semantics,description,position}settings{key,value}}}}"
+        `{
```

Comment:
doet de formatter dit zelf? dat zou geweldig zijn!

Reply:
Het doet een hoop, echter deze aanpassing is handmatig gedaan.